File: module-user/src/main/java/com/drz/user/UserModuleInit.java
package com.drz.user;

import com.drz.base.base.BaseApplication;
import com.drz.common.IModuleInit;

/**
 * 应用模块:
 * <p>
 * 类描述:
 * <p>
 *
 * @author darryrzhoong
 * @since 2020-02-29
 */
public class UserModuleInit implements IModuleInit {
    @Override
    public boolean onInitAhead(BaseApplication application) {
        return false;
    }

    @Override
    public boolean onInitLow(BaseApplication application) {
        return false;
    }
}

File: module-user/src/main/java/com/drz/user/UserModuleInit.kt
package com.drz.user

import com.drz.base.base.BaseApplication
import com.drz.common.IModuleInit

class UserModuleInit : IModuleInit {
    override fun onInitAhead(application: BaseApplication?): Boolean = true
    override fun onInitLow(application: BaseApplication?): Boolean = true
}
